### PR TITLE
Update helm task test to use same repo

### PR DESCRIPTION
**Pull request by pivotal-nader-ziada**
depends on PR #179 

remove use of external repo to make changes to test easier in the future

### CODE REVIEW

**Comment by tejal29** [comment]
looks like the test failed.

**Comment by pivotal-nader-ziada** [comment]
Needs pr #179 to be merged in first for this test to pass

File: test/helm_task_test.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func getGoHelloworldGitResource(namespace string) *v1alpha1.PipelineResource {
 			Params: []v1alpha1.Param{
 				v1alpha1.Param{
 					Name:  "Url",
-					Value: "https://github.com/pivotal-nader-ziada/gohelloworld",
+					Value: "https://github.com/knative/build-pipeline",
```
Comment (bobcatfish) [inline test/helm_task_test.go]: for the record, this is going to be pretty weird, and maybe confusing b/c it'll be pulling from master XD aw well

Comment (pivotal-nader-ziada) [reply test/helm_task_test.go]: I open to changing it, but not sure to what? :) we got comments about the external repo. 

Is it an option to get a helloworld repo in the knative org for testing?

Comment (bobcatfish) [reply test/helm_task_test.go]: it's fine to leave it for now, seems like there are cons to all the options

**Pull request merged**